jobserver/settings.py

```diff
@@ -215,3 +216,9 @@
 
 # Sentry
 initialise_sentry()
+
+
```

Comment:
Could you stick a comment (similar to the [third party comment](https://github.com/opensafely-core/job-server/blob/66f4f5dc403eab428268a46e5258cdd158effa0d/jobserver/settings.py)) to the effect of `# PROJECT SETTINGS`.

(testing out the suggestions button for the first time!)

```suggestion

# PROJECT SETTINGS

```